graphql/data/schema.js

```diff
@@ -693,10 +709,189 @@ const userType = new GraphQLObjectType({
       type: new GraphQLNonNull(GraphQLBoolean),
       description: 'whether a v4 user has been introduced to squads in the ui',
     },
+    cause: {
+      type: causeType,
+      description: 'cause type for the user',
+      args: {
+        causeId: { type: new GraphQLNonNull(GraphQLString) },
+      },
+      resolve: (_, { causeId }) => CauseModel.get(causeOverride, causeId),
+    },
   }),
   interfaces: [nodeInterface],
 })
 
+const squadCopyType = new GraphQLObjectType({
+  name: 'SquadCopy',
+  description: 'Copy for squad related widgets',
+  fields: () => ({
+    squadCounterText: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: 'Copy for SquadCounter for normal case',
+    },
+    currentMissionSummary: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: 'Copy for CurrentMission summary',
+    },
+    currentMissionDetails: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: 'Copy for CurrentMission details',
+    },
+    currentMissionAlert: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: 'Copy for CurrentMission alert',
+    },
+    currentMissionStep2: {
```

Comment:
Can you provide a more specific description for these? Confused as to what this actually does. Should there be a step1?